src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -84,7 +84,7 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// <summary>
         /// An optional client that can be used to communicate with Azure App Configuration. If provided, the connection string property will be ignored.
         /// </summary>
-        internal ConfigurationClient Client { get; set; }
+        internal IConfigurationClient Client { get; set; }
```

Comment:
Is this property used by anyone? Looks like the AACProvider has a similar property as well. Can we remove this if it's not necessary?